src/controllers/authentication.js

```diff
@@ -28,14 +28,14 @@ const createUser = async (req, res) => {
       'you can now verify your account',
       url
     );
-    const { password, ...data } = user.toJSON();
+
     return res.status(201).json({
       message: 'Account created successfully',
       data: user,
       token,
     });
   } catch (error) {
-    return res.status(500).json(error);
+    return res.status(500).json(error.message);
   }
```

Comment:
Make sure that you are not sending error object, sometimes it's not formatted an meangless to the user